studies/models.py

```diff
@@ -60,10 +60,23 @@ def __init__(self, *args, **kwargs):
             before_state_change='check_permission',
             after_state_change='_finalize_state_change'
         )
+        self.__monitoring_fields = ['structure', 'name', 'short_description', 'long_description', 'criteria', 'duration', 'contact_info', 'max_age', 'min_age', 'image', 'exit_url']
+        for field in self.__monitoring_fields:
+            setattr(self, '__original_%s' % field, getattr(self, field))
 
     def __str__(self):
         return f'<Study: {self.name}>'
 
+    def important_fields_changed(self):
+        """
+        Check if fields in self.__monitoring fields have changed.
+        """
+        for field in self.__monitoring_fields:
+            orig = '__original_%s' % field
```

Comment:
I had no idea this was there. We use django-dirtyfields in the osf, which does basically the same thing, but it feels like overkill compared to your implementation. 

I would probably change this to be the new python 3.6 string formatting `f'__original_{field}'`or at least `'__original_{}'.format(field)`. I think the `%` formatting has been frowned upon for a while. 

You also probably don't need the intermediate variable `orig` just do the interpolation in the getattr since it's only used once. 

Same things with the `__init__` usage.